Lucidum/README.md

```diff
@@ -30,11 +30,26 @@ _sourceCategory = Lucidum
 ```
 ## Collect Logs for Lucidum
 
+### Collector URL
+https://endpoint4.collection.sumologic.com/receiver/v1/http/ZaVn************8abx9m9EAuUHJQ==
+
+### cURL Examples
+The cURL examples below point to the SumoLogic endpoint set up on 12/17/2021 to start developing Lucidum support for SumoLogic.
+
+### POST example
```

Comment:
This is a manual post is there no way to send logs continuously via webhooks or installed collector local file source or S3 ?  It should as detailed similar to https://github.com/SumoLogic/sumologic-public-partner-apps/tree/master/Gigamon_ThreatINSIGHT#collect-logs-for-gigamon_threatinsight